merkle/src/store.rs

```diff
@@ -0,0 +1,411 @@
+use merkle::Element;
```

Comment:
nit for future PRs: if possible try to encapsulate no-logic-change refactorings like moving code to a new file in a separate commit to help focus the review